.circleci/src/workflows/commit.yml

```diff
@@ -1,25 +1,49 @@
 unless: << pipeline.parameters.api_triggered >>
-jobs: # TODO only build changed projects
-  - web_build
-  - web_deliver:
+jobs:
+  - build_web:
+      context:
+        - slack-app
+  - deliver_web:
       filters:
         branches:
           only:
             - main
+      context:
+        - slack-app
       production_delivery: false
       requires:
-        - web_build
+        - build_web
 
-  - native_prepare_build
-  - native_build_android:
+  - check_native:
+      context:
+        - slack-app
+
+  - build_android:
+      context:
```

Comment:
Cool that it is possible to have the context here! Maybe it is a duplication though in other workflows. As long as we only have one set of contexts we should probably keep them in the job files.

Reply:
There are no contexts on job level :)

Reply:
https://circleci.com/docs/2.0/contexts/